src/js/disability-benefits/526EZ/helpers.jsx

```diff
@@ -117,3 +137,125 @@ export const privateMedicalRecordsIntro = ({ formData }) => {
+  }
+
+  validatePostalCodes(errors, formData);
+}
+
+export const recordReleaseWarning = (
+  <div className="usa-alert usa-alert-warning no-background-image">
+    <span>Limiting consent means that your doctor can only share records that are directly related to your condition. This could add to the time it takes to get your private medical records.</span>
+  </div>
+);
+
+export const documentDescription = () => {
+  return (
+    <div>
+      <p>File upload guidelines:</p>
+      <ul>
+        <li>File types you can upload: .pdf, .jpeg, or .png</li>
+        <li>Maximum file size: 50 MB</li>
+      </ul>
+      <p><em>Large files can be more difficult to upload with a slow Internet connection</em></p>
+    </div>
+  );
+};
+
+export const additionalDocumentDescription = () => {
+  return (
+    <div>
+      <p>If you have other evidence, like lay or buddy statements, that you would like to submit, you can upload them here.</p>
+      <p>File upload guidelines:</p>
+      <ul>
+        <li>File types you can upload: .pdf, .jpeg, or .png</li>
+        <li>Maximum file size: 50 MB</li>
+      </ul>
+      <p><em>Large files can be more difficult to upload with a slow Internet connection</em></p>
+    </div>
+  );
+};
+
+const documentLabels = {
+  1: 'Discharge',
+  2: 'Marriage related',
+  3: 'Dependent related',
+  // 4: 'VA preneed form',
+  5: 'Letter',
+  6: 'Other'
+};
+
+const getVACenterName = (center) => center.treatment.treatmentCenterName;
+const getPrivateCenterName = (release) => release.privateRecordRelease.treatmentCenterName;
+
+const listifyCenters = (center, idx, list) => {
+  const getName = center.treatment ? getVACenterName : getPrivateCenterName;
+  const notLast = idx < (list.length - 1);
+  const justOne = list.length === 1;
+  const atLeastThree = list.length > 2;
+  return (
+    <strong key={idx}>
+      {`${notLast || justOne ? '' : ' and '}${getName(center)}${atLeastThree && notLast ? ', ' : ''}`}</strong>
+  );
+};
+
+
+export const evidenceSummaryView = ({ formData }) => {
+  const { treatments: VATreatments, privateRecordReleases, privateRecords, additionalDocuments } = formData;
+  return (
+    <div>
+      <ul>
+        {VATreatments &&
+        <li>We will get your medical records from {VATreatments.map(listifyCenters)}.</li>}
+        {privateRecordReleases &&
+        <li>We will get your private medical records from {privateRecordReleases.map(listifyCenters)}.</li>}
+        {privateRecords && <li>We have the private medical records you uploaded.</li>}
+        {additionalDocuments &&
+        <li>We have this additional evidence that you uploaded:
+          <ul>
+            {additionalDocuments.map((document, id) => {
+              return (<li className="dashed" key={id}>
+                <strong>{`${documentLabels[document.attachmentId]} (${document.name})`}</strong>
+              </li>);
+            })
+            }
+          </ul>
+        </li>}
+      </ul>
+    </div>
+  );
```

Comment:
Kindly focus on the `evidenceSummaryView ` and `listifyCenters` as that is the updated portion of this file in this PR, other changes refer to other PRs that this is branched off of (and will not be displayed here once those are merged in).